ConfigurationSystem/Client/Utilities.py

```diff
@@ -652,3 +652,8 @@ def getElasticDBParameters(fullname):
   parameters['SSL'] = ssl
 
   return S_OK(parameters)
+
+
+def getOAuthAPI():
```

Comment:
unused.
Also hardcode the instance (`Production`)

Reply:
fixed